database/migrations/default/1728576888683_add_apd_sectors/down.sql

```diff
@@ -0,0 +1,2 @@
+drop trigger if exists set_updated_at_apd_sectores on apd_sectors;
```

Comment:
@johnclary There is a small typo in the trigger name (sectores), but it made me wonder if this trigger needs to be dropped in the down migration (also in `database/migrations/default/1728415316165_add_zip_codes/down.sql`) since all these tables had `updated_at` removed or updated to `created_at`?

Reply:
Nice catch, these triggers dont exist anyore, thats the issue, right?

Reply:
yes, nice catch! this is a great example of the perils of using `IF EXISTS`.

fixed ✅